src/OpenLoco/src/Vehicles/Orders.cpp

```diff
@@ -206,6 +206,25 @@ namespace OpenLoco::Vehicles
         return begin();
     }
 
+    uint8_t swapAdjacentOrders(Order& a, Order& b)
```

Comment:
I'd like this to go into the `OrderManager` namespace. This becomes feasible after #1978 has been merged.